mmdet/datasets/openimages.py

```diff
@@ -786,8 +825,8 @@ def get_relation_matrix(self, hierarchy_file):
             relationship between the parent class and the child class,
             of shape (class_num, class_num).
         """
-
-        assert hierarchy_file.endswith('np')
+        if self.assert_suffix:
+            assert hierarchy_file.endswith('np')
```

Comment:
This might not be compatible with ceph, consider convert it to pkl format.

Reply:
this is the official format, have checked. This can work